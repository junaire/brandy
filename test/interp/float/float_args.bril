# ARGS: 3 3.0 -0.3
@main(float_one : float, float_two : float, float_three : float) {
  print float_one float_two float_three;
}