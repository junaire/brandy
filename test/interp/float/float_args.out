3.00000000000000000 3.00000000000000000 -0.29999999999999999
